node/src/BaseClient.ts

```diff
@@ -4434,6 +4434,8 @@ export class BaseClient {
      *           of indices that represent the location of the common subsequences in the strings held
      *           by `key1` and `key2`.
      *
+     *     See example for more details.
```

Comment:
Is this actually necessary when the example is right below?  This feels a bit redundant.

Reply:
I agree, but I don't know how docs would be rendered by `javadoc`, `doxygen` or other tool.
For example IDEA built-in renderrer does not render example block on classes.